src/wallet.rs

```diff
@@ -452,6 +454,20 @@ pub fn process_command(config: &WalletConfig) -> Result<String, Box<error::Error
                 "programId": format!("{}", program.pubkey()),
             }).to_string())
         }
+        WalletCommand::GetLeaderReadiness => {
+            // TODO: This logic is currently only correct for initial network boot;
+            // needs updating for leader rotation
+            let transaction_count = RpcRequest::GetTransactionCount
```

Comment:
Yes, that's an option. If there are other potential future client uses for transaction count, that would be a better/simpler solution. Are there? I was thinking that there would be greater need for clients to be able to test network readiness, and it would make sense for us to handle that logic.

Reply:
I guess from the RPC API POV, we already expose GetTransactionCount so reflecting that through the wallet seems nice regardless.   Then once we do that, we have what we need to easily compute our current meaning of "network readiness" (> 0 transactions).  We can always builds a more elaborate  "network readiness" in the future when we have additional requirements.

Reply:
I see solana-wallet as an additional layer of abstraction, like the JS api, with its whole point to do things like compute our current meaning of network readiness. But I'm not willing to die on that hill :)
I'll plumb GetTransactionCount through solana-wallet for now, and reapproach network readiness later on.

Reply:
If you spot anything else that thin_client is exposing that jsonrpc is not, drag that along as well please.

Reply:
@garious, do you mean anything else that jsonrpc is exposing that solana-wallet is not?